blockchain/processing.py

```diff
@@ -676,6 +676,9 @@ def _execute_timestamping(self, config):
         # takes the list of hashes to be transmitted and hashes with 256 bit to get in form to send
         transaction_hash = final_hash(hashes, type=256)
 
+        # normal SHA512 hash for all lower VR contents
+        lower_hash = final_hash(hashes)
```

Comment:
FYI  - added this to do the consistent SHA512 hashing on all lower VR contents of this phase 5 VR.
:)